### ISSUE TITLE
[ALEXA] Cannot controlOnOff Error: Won't control zigbee.0.xxxx.color as we have a switch available

### ISSUE THREAD

**Issue by AdamK-DE** [ioBroker/ioBroker.iot#651]
seit ioT 2.0.11 beata ist mein log voll mit:

**Comment by AdamK-DE** [comment]
ps: ließt man im www nach dann brauch ioT doch keinen Aus/An state bei birnen?! überall steht: alexa regelt das nur per brightness... klappt theoretisch auch so... praktisch hängt es da bei ioT aber irgendwo.. 

per blockly bediene ich die immer so und klappt:

AN: erst brightness auf min 1 setzen
color change etc/irgendwas
bri hoch

AUS: bri wieder auf 0

bei ioT bei AN "letzter Wert" klappt leider genau so wenig wie "25%" etc.. bzw alexa licht an lässt die birne dann 1x kurz aufblinken und dann ist sie wieder aus. erst "alexa helligkeit auf 20%" etc macht die wieder an.. 

wollte eben die datenpunkte dazu schreiben aber siehe da: alles klappt gerade per alexa/ioT argh.......... keine ahnung wieso es nun läuft.. war aber schon paar mal so beim testen.. licht an/aus oder lampe einzeln geht gerade... normalerweise geht das nie wenn letzte brightness auch zu niedrig war. schaltet man bei 10% aus dann geht die nie wieder an ohne das man brightness nicht direkt höher setzt..........

**Comment by Bibo13** [comment]
Nach dem Update von IOT 2.0.11 stable habe ich das Gleiche verhalten. Mein Log wird mit Warnungen überschwemmt.
Gibt es da mittlerweie eine Lösung?

**Comment by Diginix** [comment]
Bei mir das gleiche. Selbst wenn ich den switch State nicht mit ins iot Device nehme, bekomme ich die warnung. Habe GU10 Spots mit colortemp und brightness und nur diese 2 Objekte in einem iot Device. Egal ob ich das Gerät An/Ausschalte, die Helligkeit oder Farbtemp. steuere, es gibt immer 1..n Warnungen.

**Comment by Joko2022** [comment]
Beimir kommt die Warnung "[ALEXA] Cannot controlOnOff Error: Won't control zigbee.0.group_1.state as we have a switch available" wenn ich versuche, die Leuchte (Eglo 900024) über Alexa auszuschalten. Die Helligkeit geht dann auf  ca. 5 %. Direktes Ausschalten über den Datenpunkt state oder per Blockly oder per App funktioniert einwandfrei. Das iot-Device ist genauso angelegt, wie 5 andere RGB-Leuchten. Die schalten einwandfrei über Alexa.
Weiterhin kommen die Warnungen für die anderen Datenpunkte im iot-Device - color, colortemp und transition time. Für brightness kommt keine Warnung

**Comment by Grmlrckr** [comment]
Bei mir kommt die gleiche Meldung. Bei ein/aus, aber auch bei Farbwechsel. Dabei ist es egal ob original HUE-Lightstrips oder Lidl Livarno Lux GU10 Lampen.

**Comment by Negalein** [comment]
Dito, bei mir auch

```Heute 15:28	warn	iot.1	[ALEXA] Cannot controlOnOff Error: Won't control hue.0.Kinderzimmer.sat as we have a switch available
Heute 15:28	warn	iot.1	[ALEXA] Cannot controlOnOff Error: Won't control hue.0.Kinderzimmer.on as we have a switch available
Heute 15:28	warn	iot.1	[ALEXA] Cannot controlOnOff Error: Won't control hue.0.Kinderzimmer.hue as we have a switch available
Heute 15:28	warn	iot.1	[ALEXA] Cannot controlOnOff Error: Won't control hue.0.Kinderzimmer.ct as we have a switch available```

**Comment by fn-x** [comment]
👍 Bei mir auch.

**Comment by Bongo2** [comment]
Dann sag ich's auch: Bei mir auch :-)

**Comment by fn-x** [comment]
Ich habe die Smart Devices mal alle auf "keinen Wert" gestellt. Dann wird wieder der Switch DP genutzt. Das Log Level habe ich erst Mal auf Error gestellt.

**Comment by Bongo2** [comment]
Ich habe bisher die Warnungen über mich ergehen lassen. Aber es wäre schon schön wenn das Log normal aussieht.
Wie muss jetzt das Gerät (Lampe) im iot aussehen, damit es ohne Warnung funktioniert.

**Comment by Diginix** [comment]
> Ich habe bisher die Warnungen über mich ergehen lassen. Aber es wäre schon schön wenn das Log normal aussieht. Wie muss jetzt das Gerät (Lampe) im iot aussehen, damit es ohne Warnung funktioniert.

Gibt kein Setup was ohne Fehlermeldung kommt. Außer vllt du nimmst nur den switch state. Aber dann kannst du keine Helligkeit steuern und auch keine colortemp.

**Comment by Grmlrckr** [comment]
> Gibt kein Setup was ohne Fehlermeldung kommt. Außer vllt du nimmst nur den switch state. Aber dann kannst du keine Helligkeit steuern und auch keine colortemp.

Naja, bis vor ein paar Monaten gabs die Fehlermeldungen ja auch nicht. Also muss es ja irgendeine Änderung gegeben haben.

**Comment by Diginix** [comment]
Mag sein, aber dann nicht im Setup der iot Devices sondern im iot Adapter oder der Amazon API. Beides kannst du als Anwender aber aktuell nicht durch die Gerätekonfiguration beeinflussen. Sobald es ein Gerät mit mehr als an/aus (boolean switch) ist, also dimmbar, Farbe bzw. Farbtemperatur steuerbar ist, musst du diese states ja an Amazon durchreiche und dann kommt zwangsläufig diese Fehlermeldung.

**Comment by Grmlrckr** [comment]
Nein, schon klar. Das ist ja der Punkt. Als Anwender kann ich da nichts machen. Deshalb ja die Hoffnung, dass da mal was passiert.

**Comment by Diginix** [comment]
Vermutlich ist Ingo aktuell zu sehr mit anderem beschäftigt (Matter uzsw.). Es gibt ja schon die v3 API, die auch viel mehr Gerätetypen unterstützt, aber mWn auch noch in den Kinderschuhen steckt. Lösbar ist das auf alle Fälle, aber es braucht einen Freiwilligen mit Zeit ;-)
